core/src/main/java/org/infinispan/container/entries/ReadCommittedEntry.java

```diff
@@ -321,4 +327,31 @@ public boolean undelete(boolean doUndelete) {
       return false;
    }
 
+   public static class Externalizer extends AbstractExternalizer<ReadCommittedEntry> {
```

Comment:
The objective is not to make the test easier. A user could easily decide to return the view out of the lambda. I don't think that's very far fetched really. Sure I could do the asserts directly the lambda, but that's not the point here.

I will however look into a way to avoid the need to marshall ReadCommittedEntry itself.